Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:focal as builder
+LABEL maintainer "who_should_this_be@gmail.com"
+LABEL description="This image contains the cargo contract command"
+
+ARG RUSTC_VERSION="nightly-2020-10-27"
+ENV RUSTC_VERSION=$RUSTC_VERSION
+ENV PROFILE=release
+ENV PACKAGE=polkadot-runtime
+
+RUN mkdir -p /cargo-home /rustup-home
```

Comment:
please glue all the `RUN` steps together, it won't hurt and will make fewer layers in an image.